source/logrepl/internal/publication_test.go

```diff
@@ -94,59 +99,6 @@ func TestCreatePublicationForTables(t *testing.T) {
 	}
 }
 
-func TestCreatePublicationForTablesAndAllTables(t *testing.T) {
-	ctx := context.Background()
-	is := is.New(t)
-	pub := test.RandomIdentifier(t)
-	conn := test.ConnectSimple(ctx, t, test.RegularConnString)
-
-	err := CreatePublication(
-		ctx,
-		conn.PgConn(),
-		pub,
-		CreatePublicationOptions{
-			Tables:    []string{"something"},
-			AllTables: true,
-		},
-	)
-
-	is.True(err != nil)
-	_, ok := err.(*pgconn.PgError)
-	is.True(!ok) // err shouldn't be a PgError, no statement should be even sent to Postgres
-}
-
-func TestCreatePublicationAllTables(t *testing.T) {
```

Comment:
Did we remove support for creating a publication for all tables? The connector config [advertises](https://github.com/ConduitIO/conduit-connector-postgres/blob/10e4575daf336ec9bea04dd6970e2f429667d901/source/config.go#L55) this functionality.

Reply:
The refactor which happened earlier, which reads all tables, does not communicate to the iterator that it should create a publication on all tables, but rather it provides a slice of tables (as per getAllTables). 

The solution for this will be to update the publication with the new table list. As per this https://github.com/ConduitIO/conduit-connector-postgres/issues/145

Reply:
I see, so we manually fetch a list of all tables and populate the config field. All good 👍